### PR TITLE
Sea Shanties - /tg/'s singing update

### PR DESCRIPTION
**Pull request by Cenrus**
https://github.com/tgstation/tgstation/pull/49063/

## About this Pull Request

image

Adds singing to space. Prefix your message with % to sing a tune with your fellow crewmembers.

## Changelog

:cl: EOBGames
add: added Singing. Simply prefix your message with % to start singing some sea shanties.
/:cl:

### CODE REVIEW

**Comment by Karmic-Skink** [comment]
Needs //NSV13 tags on core file changes to be trackable for rebasing.

**Comment by Cenrus** [comment]
https://github.com/BeeStation/BeeStation-Hornet/pull/1204

Opened a PR upstream too.3

File: code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -197,6 +197,15 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	if(language)
 		var/datum/language/L = GLOB.language_datum_instances[language]
 		spans |= L.spans
+	
+	if(message_mode == MODE_SING) //nsv13 start
+	#if DM_VERSION < 513
+		var/randomnote = "~"
+	#else
+		var/randomnote = pick("\u2669", "\u266A", "\u266B")
+	#endif
```
Comment (covertcorvid) [inline code/modules/mob/living/say.dm]: Do these need to be indented one more?

**Comment by francinum** [comment]
Made redundant by #396

**Comment by Cenrus** [comment]
Merging from upstream.

**Pull request closed without merging**